### PR TITLE
Yda 5575 fileviewer

### PR DESCRIPTION
**Pull request by claravox**

### CODE REVIEW

**Comment by claravox** [comment]
I have got 2 failing api tests: test_vault_meta_form_save_in_vault, test_vault_submit
I don't think they are related to my code?

**Comment by lwesterhof** [comment]
> I have got 2 failing api tests: test_vault_meta_form_save_in_vault, test_vault_submit I don't think they are related to my code?

No, I don't think so. I will check it.

File: static/lib/highlight-js-11.9.0/highlight.js
Comment (claravox) [inline static/lib/highlight-js-11.9.0/highlight.js:1]: Highlight js should not have problems with injection attacks:
```
We escape all 5 characters that are the least bit problematic for HTML.  We need the raw source code as input.
```
https://github.com/highlightjs/highlight.js/issues/3229

File: static/lib/highlight-js-11.9.0/languages/accesslog.js
Comment (claravox) [inline static/lib/highlight-js-11.9.0/languages/accesslog.js:1]: Highlight js on their website lets you select which languages to download before downloading:
https://highlightjs.org/download

I selected the file formats of "scientific" and "common".

File: static/lib/highlight-js-11.9.0/styles/a11y-dark.css
Comment (claravox) [inline static/lib/highlight-js-11.9.0/styles/a11y-dark.css:1]: Since we only use 2 of the themes we could probably delete the other ones?

Comment (lwesterhof) [reply static/lib/highlight-js-11.9.0/styles/a11y-dark.css:1]: Agree

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**